cypress/page-objects/basePage.js

```diff
@@ -0,0 +1,38 @@
+export class BasePage {
+
+  get homeCTA() {
+    return cy.get('.sc-kstrdz a[href="/"]');
+  }
+
+  get searchBox() {
+    return cy.get('input[type="search"]');
+  }
+
+  focusInSearchBox() {
+    this.searchBox.click();
+  }
+
+  get shopCTA() {
+    return cy.get('.sc-fFubgz[href="/shop"]');
+  }
+
+  get contactCTA() {
```

Comment:
getters first, then class methods, lets please do not make mess in PO files

Reply:
done